agents/triage_agent.py

```diff
@@ -291,9 +292,11 @@ class TriageState(BaseModel):
     cve_eligibility_result: CVEEligibilityResult | None = Field(default=None)
     triage_result: OutputSchema | None = Field(default=None)
     target_branch: str | None = Field(default=None)
+    metrics: dict | None = Field(default=None)
 
 
 async def run_workflow(jira_issue, dry_run):
+    current_metrics_middleware = MetricsMiddleware()
```

Comment:
I don't think this belongs here. It should be either injected (added post-construction, before running), or it should be an argument to the constructor/workflow. Other agent modules have functions such as `create_rebase_agent()`, that could be used here (and vice versa, the move of `run_workflow()` to the top level could be done in the other agent modules as well).